docs/reference/system/users.md

```diff
@@ -409,25 +420,45 @@ POST /graphql/system
 
 ```graphql
 type Mutation {
-	create_users_items(data: [create_directus_users_input!]!): [directus_users]
+  create_users_items(data: [create_directus_users_input!]!): [directus_users]
 }
 ```
 
 ##### Example
 
 ```graphql
 mutation {
-	create_users_items(
-		data: [
-			{ email: "admin@example.com", password: "p455w0rd", role: "c86c2761-65d3-43c3-897f-6f74ad6a5bd7" }
-			{ email: "another@example.com", password: "d1r3ctu5", role: "c86c2761-65d3-43c3-897f-6f74ad6a5bd7" }
-		]
-	) {
-		email
-		role
-	}
-}
-```
+  create_users_items(
+    data: [
+      {
+        email: "admin@example.com"
+        password: "p455w0rd"
+        role: {
+          id: "c86c2761-65d3-43c3-897f-6f74ad6a5bd7"
+          name: "Public"
+          admin_access: false
+          enforce_tfa: false
+        }
+      }
+      {
+        email: "another@example.com"
+        password: "d1r3ctu5"
+        role: {
+          id: "c86c2761-65d3-43c3-897f-6f74ad6a5bd7"
+          name: "Public"
+          admin_access: false
+          enforce_tfa: false
+        }
+      }
+    ]
+  ) {
+    email
+    role
+  }
+}
+```
+
+Please note that if you include the Role in the `create_users_items` call it will be treated as an Upsert and not only as adding a relationship. So make sure the ID exists, and the other parameters match the existing role, otherwise it could be modified by the user call.
```

Comment:
It looks like your editor might've automatically reformatted everything else on this page 🤔 Mind reverting that so this PR is only the intended change? Thanks!

Reply:
Hmm interesting, wonder how that happend because we've disabled formatting of code blocks in our prettier config 🤔

Reply:
Who knows! Maybe they're using another formatter, or have their editor configured with a custom set of rules. Anything is possible!